### PR TITLE
Enables generation of inflection tables for new locale 'xx'. 

### PR DESCRIPTION
**Pull request by smkvr98**
Deals with substep 2 of step 1 of [issue 12](https://github.com/digitalpalitools/web-ui/issues/12) - `calls to pls will take in a lcid parameter
test with lcid = xx for now, actual string will be '^' + en string + '$' (e.g. opt => ^optative$)
the en string is taken from from 'description' column of abbreviations table`.

### CODE REVIEW

File: pls_core/src/inflections/snapshots/pls_core__inflections__tests__inflection_tests__declension_5_only_x_gender_deva.snap [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+source: pls_core/src/inflections/mod.rs
+expression: html
+
+---
+no such column: deva
```
Comment (parthopdas) [inline pls_core/src/inflections/snapshots/pls_core__inflections__tests__inflection_tests__declension_5_only_x_gender_deva.snap]: this is a sporious test i.e. dead code. fine for this pushing in this commit but please dont deliberately commit dead code.

File: pls_core/src/inflections/mod.rs [hunk 2]
```diff
@@ -59,11 +60,11 @@ pub fn generate_inflection_table(
     host_version: &str,
     transliterate: fn(&str) -> Result<String, String>,
     exec_sql_query: fn(&str) -> Result<String, String>,
+    locale: &str,
 ) -> Result<String, String> {
     let q = SqlQuery::new(exec_sql_query);
     let pm = get_pali1_metadata(pali1, transliterate, &q)?;
-    let body = generators::create_html_body(&pm, transliterate, &q)?;
-
```
Comment (parthopdas) [inline pls_core/src/inflections/mod.rs:66]: these are unnecessary line deletions. not related to your change so should not be there.

if there is a reason for this, please have a separate refactoring commit

File: pls_core/src/inflections/generators/declension_pron_x.rs [hunk 3]
```diff
@@ -37,17 +39,19 @@ pub fn create_html_body(
     stem: &str,
     transliterate: fn(&str) -> Result<String, String>,
     q: &SqlQuery,
+    locale: &str,
 ) -> Result<String, String> {
     let table_name = &generators::get_table_name_from_pattern(pattern);
     let view_models = create_case_view_models(&pron_type, &table_name, transliterate, &q, &stem)?;
     let in_comps_inflections = Vec::new();
-
```
Comment (parthopdas) [inline pls_core/src/inflections/generators/declension_pron_x.rs:44]: same here - no unnecessary change should be there.

**Review by parthopdas — APPROVED** [review]

File: pls_core/src/inflections/mod.rs (hunk 2 above)
Comment (smkvr98) [reply pls_core/src/inflections/mod.rs:66]: I'll keep this in mind.
